### PR TITLE
netlink-packet-route: implement VRF support

### PR DESCRIPTION
**Pull request by ffmancera**
VRF interfaces only has table information. This patch is introducing
InfoVrf nla.

Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

**Comment by gabrik** [comment]
@ffmancera please do a run of `cargo clippy` and `cargo fmt`

**Comment by ffmancera** [comment]
> @ffmancera please do a run of `cargo clippy` and `cargo fmt`

Done!

File: netlink-packet-route/src/rtnl/link/nlas/link_infos.rs [hunk 1]
```diff
@@ -1345,6 +1355,54 @@ impl<'a, T: AsRef<[u8]> + ?Sized> Parseable<NlaBuffer<&'a T>> for InfoIpVlan {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum InfoVrf {
+    Unspec(Vec<u8>),
```
Comment (cathay4t) [inline netlink-packet-route/src/rtnl/link/nlas/link_infos.rs:1360]: The kernel never use `Unspec`, if that really happens, it should fit into `Other`.

**Review by cathay4t — CHANGES_REQUESTED** [review]

File: netlink-packet-route/src/rtnl/link/nlas/link_infos.rs (hunk 1 above)
Comment (ffmancera) [reply netlink-packet-route/src/rtnl/link/nlas/link_infos.rs:1360]: I think we discussed this before. In that case, we should remove `Unspec` from the others Info* to keep consistency.

**Review by gabrik — APPROVED** [review]

**Pull request merged**